server/src/models/results/loginResult.ts

```diff
@@ -0,0 +1,12 @@
+import { SocoboUser } from "./../database/socoboUser";
```

Comment:
Maybe I would replace Result with something like Response since these kind of object is used as response. So loginResult would become loginResponse or maybe loginResponseBody to be more concrete